README.rst

```diff
@@ -42,8 +42,35 @@ The created venv should be located somewhere around .tox/dev/Scripts.
 
 Versioning
 ==========
-To track the versions of this python package we use `setuptools-scm <https://pypi.org/project/setuptools-scm/>`_.
-There are different kind of ways to achieve the version tracking. We will use the `pyproject.toml` usage.
+Short background information about versioning of python packages.
+At the moment (10.02.2021) there are `**seven** ways to define the version of your package <https://packaging.python.org/guides/single-sourcing-package-version/>`_.
+We use `setuptools-scm <https://pypi.org/project/setuptools-scm/>`_ for versioning so we can use the labels of git to define the version.
+The tool itself again has several ways how to configure it.
+We will use the `pyproject.toml` file to configure setuptools-scm.
+There we tell the build-system with `"setuptools_scm[toml]>=3.4"` that we use setuptools_scm and the version must be at least `3.4`.
+The `[toml]` section tells setuptools-scm that it finds all settings in our pyproject.toml file.
+`[tool.setuptools_scm]` in pyproject.toml enables version inference.
+In the setup.py we have to use the attribute `use_scm_version=True`.
+
+To create the version number itself, we stick to the default behavior of setuptools-scm.
+It will take a look at three things:
+
+1. latest tag (with a version number)
+2. the distance to this tag (e.g. number of revisions since latest tag)
+3. workdir state (e.g. uncommitted changes since latest tag)
+
+and uses roughly the following logic to render the version:
+
+**no distance and clean:**
```

Comment:
Wolltest du hier eine Aufzählung? Und sollten die wie sollten die Sachen innerhalb der `{ }` aussehen? Im Moment schaut es so in der Vorschau aus: 
![grafik](https://user-images.githubusercontent.com/73470827/107521338-b9c55480-6bb2-11eb-8ca7-f55b7cd7dd77.png)